sources/osgShadow/shaders/shadowLinearSoft.glsl

```diff
@@ -1,36 +1,48 @@
 #pragma include "floatFromTex.glsl"
 
-
-#ifdef _OUT_DISTANCE
-#define OPT_ARG_outDistance ,out float outDistance
-#else
-#define OPT_ARG_outDistance
-#endif
-
 // simulation of texture2Dshadow glsl call on HW
 // http://codeflow.org/entries/2013/feb/15/soft-shadow-mapping/
-float texture2DCompare(const in sampler2D depths, const in vec2 uv, const in float compare, const in vec4 clampDimension){
+float texture2DCompare(const in sampler2D depths,
+                       const in vec2 uv,
+                       const in float compare,
+                       const in vec4 clampDimension){
     float depth = getSingleFloatFromTex(depths, clamp(uv, clampDimension.xy, clampDimension.zw));
     return compare - depth;
 }
 
+#ifdef _JITTER_OFFSET
+
+#define INT_SCALE3 vec3(.1031, .1030, .0973)
+#define _rand3out(_p) _p += dot(_p, _p.yzx + 19.19); return fract((_p.xxy + _p.yzz) * _p.zyx);
```

Comment:
won't need `_rand3out` just `_rand2out`

Also the names of these functions might collide with rand.glsl in sketchfab so not great.
Maybe moving the rand.glsl in osgjs? (or at least renaming these rand in osgjs)